proto-public/pbdataplane/dataplane.proto

```diff
@@ -76,16 +78,27 @@ enum ServiceKind {
 }
 
 message GetEnvoyBootstrapParamsResponse {
-  ServiceKind service_kind = 1;
-  // service is be used to identify the service (as the local cluster name and
+  reserved 1;
+
+  // deprecated: use cluster_name instead.
+  // service is used to identify the service (as the local cluster name and
   // in metric tags). If the service is a connect proxy it will be the name of
   // the proxy's destination service, for gateways it will be the gateway
   // service's name.
-  string service = 2;
+  string service = 2 [deprecated = true];
+
+  // cluster_name is used to identify this proxy (as the local cluster name and
+  // in metric tags). For v1, this should be the service name.
+  // If the service is a connect proxy it will be the name of
+  // the proxy's destination service, for gateways it will be the gateway
+  // service's name.
+  // For v2, this should be the workload identity name.
+  string cluster_name = 10;
```

Comment:
I don't understand the "cluster" terminology instead of maybe "identity," but I think that's my ignorance of how the pieces go together.

Reply:
That was my initial thought as well. After reading the comment I see that this is to be used for envoy cluster names but I think it might be easier to reason about named `identity` instead.

Reply:
Yeah we can rename it to identity. My reasoning for cluster name was motivated by how Envoy calls it and it seemed generic enough to fit both v1 and v2. I do like `identity` better though in this context.